citations-tool/tool/src/webapp/css/citations.css

```diff
@@ -173,10 +173,17 @@ ol.vertical li.placeholder::before {
 .descriptionButtons {
 	padding:5px;
 }
+ol.serialization li div.sectionEditor:hover {
+	cursor: pointer;
+	background: #2a4667 none repeat scroll 0 0;
```

Comment:
Didn't we have colours coming from properties? Is this going to make it difficult to contribute back?